sourced/ml/tests/test_dump.py

```diff
@@ -77,38 +76,26 @@ def tearDown(self):
         if os.path.exists("/tmp/ml-test-dump"):
             shutil.rmtree("/tmp/ml-test-dump")
 
-    def setUp(self):
-        self.maxDiff = None
-
     def test_id2vec(self):
-        with captured_output() as (out, err, log):
+        with captured_output() as (out, _, _):
             dump_model(self._get_args(input=paths.ID2VEC))
-        self.assertEqual(err.getvalue(), "")
```

Comment:
No reason to remove it. It checks that `stderr` is empty and it should be.
Same for `log` Here and everywhere.

Reply:
(Discussed this in real time, backloging):
So I added this for debugging purpose in previous PR for modelforge requirements. Removed because: 
- tests is meant to check ours model's `dump` works, that's it
- dulwich still gives off random (but apparently non breaking) errors in `stderr` which makes build fail